modules/installation-network-user-infra.adoc

```diff
@@ -148,7 +148,7 @@ ifndef::ibm-z,azure[]
 [id="installation-host-names-dhcp-user-infra_{context}"]
 == Setting the cluster node hostnames through DHCP
 
-On {op-system-first} machines, the hostname is set through NetworkManager. By default, the machines obtain their hostname through DHCP. If the hostname is not provided by DHCP, it is obtained through a reverse DNS lookup. Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. You can avoid this by using DHCP to provide the hostname for each cluster node.
+On {op-system-first} machines, the hostname is set through NetworkManager. By default, the machines obtain their hostname through DHCP. If the hostname is not provided by DHCP or set statically through kernel arguments or another method, it is obtained through a reverse DNS lookup. Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. You can avoid this by using DHCP to provide the hostname for each cluster node.
```

Comment:
Would this make more sense to have it in like a listed format? Not sure if that does anything to the meaning.
Basically removing one "or" and adding a comma:
> 
If the hostname is not provided by DHCP, set statically through kernel arguments, or another method,

Reply:
This whole sentence just read weird to me. I don't like how vague "another method" is. But I made your change and I think it helped!

Reply:
It read weird to me too. I think your change looks great!